rust/Makefile.am

```diff
@@ -81,9 +81,8 @@ vendor:
 
 if HAVE_CBINDGEN
 gen/rust-bindings.h: $(RUST_SURICATA_LIB)
-	rm -f gen/rust-bindings.h
 	cbindgen --config $(abs_top_srcdir)/rust/cbindgen.toml \
-		--quiet --output $(abs_top_builddir)/rust/gen/rust-bindings.h
+		--quiet --verify --output $(abs_top_builddir)/rust/gen/rust-bindings.h || true
```

Comment:
Q: what does `|| true` mean here? Is it used in case files were diff?

Reply:
`cbindgen --verify` returns a non null code if files are different.
In this case, we do not want the Suricata build to fail, just go on.

It could be `|| echo rust-bindings.h was modified`

Does that answer your question ?

Reply:
Yes, thank you! :)